core/App/localization/fr/index.ts

```diff
@@ -1,98 +1,197 @@
+        "IAgree": "J'accepte",
+        "Attestation": "J'ai lu, je comprends et j'accepte les conditions."
+    },
+    "PinCreate": {
+        "UserAuthenticationPin": "NIP d'authentification de l'utilisateur",
+        "PinMustBe6DigitsInLength": "Le NIP doit comporter 6 chiffres",
+        "PinsEnteredDoNotMatch": "Les NIP saisis ne correspondent pas",
+        "6DigitPin": "NIP à 6 chiffres",
+        "ReenterPin": "Entrer le NIP à nouveau",
+        "Create": "Créer"
+    },
+    "PinEnter": {
+        "IncorrectPin": "NIP incorrect"
+    },
+    "ContactDetails": {
+        "Created": "Créé",
+        "ConnectionState": "État de la connexion",
+        "AContact": "Un contact"
+    },
+    "Credentials": {
+        "CredentialsNotFound": "Justificatifs non trouvés",
+        "CredentialDetails": "Détails des justificatifs"
+    },
+    "CredentialDetails": {
+        "Id": "Identifiant :",
+        "CreatedAt": "Créé à :",
+        "Version": "Version",
+        "Issued": "Délivré",
+        "PrivacyPolicy": "Politique sur la protection de la vie privée",
```

Comment:
Politique de confidentialité is the most common term I think

Reply:
Oui OK